crates/holochain_conductor_api/src/admin_interface.rs

```diff
@@ -585,6 +585,8 @@ pub enum ExternalApiWireError {
     RibosomeError(String),
     /// Error activating app.
     ActivateApp(String),
+    /// The zome call failed authentication.
```

Comment:
This might be a good place to point to the documentation for how to make it pass? If somebody runs into this error it'd be nice to give them a way forwards to understanding what they did wrong

Reply:
Yes, great idea.